start.sh

```
@@ -1,19 +1,18 @@
-rm $HOME/.gitconfig $HOME/.gitignore_global $HOME/.tmux.conf $HOME/.asdfrc $HOME/.bash_profile
+rm $HOME/.gitconfig $HOME/.gitignore_global $HOME/.tmux.conf $HOME/.bash_profile
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)